HipercowApi/Middleware/ExceptionHandlingMiddleware.cs

```diff
@@ -34,12 +38,13 @@ public async Task InvokeAsync(HttpContext context)
             }
             catch (Exception ex)
             {
-                _logger.LogError(ex, "An unhandled exception occurred");
-                await HandleExceptionAsync(context, ex);
+                var errorId = Guid.NewGuid().ToString();
+                _logger.LogError(ex, "Unhandled exception. Error ID: {ErrorId}", errorId);
+                await HandleExceptionAsync(context, ex, errorId);
```

Comment:
Is this still getting called for non-500s? Doesn't really need to be - could log much less info for 401s etc

Reply:
Oh, it probably still is... hmm...